src/EditorFeatures/Core/DocumentationComments/DocumentationCommentSuggestion.cs

```diff
@@ -95,6 +108,46 @@ private async Task<bool> TryDisplayProposalAsync(SuggestionSessionBase session,
             return false;
         }
 
+        public async Task DismissSuggestionSessionAsync(CancellationToken cancellationToken)
+        {
+            await DisposeAsync().ConfigureAwait(false);
+            await RunWithEnqueueActionAsync(
+                "DismissSuggestionSession",
+                async () => await _suggestionSession!.DismissAsync(ReasonForDismiss.DismissedDueToInvalidProposal, cancellationToken).ConfigureAwait(false),
+                cancellationToken).ConfigureAwait(false);
+        }
```

Comment:
why is thre DIsmiss *and* Dispose?  WHy not just one concept?

Reply:
Dismiss is on the SuggestionSession (platform owned), which is what we start to give us exclusive rights to display grey text later. Dispose is on us to allow line completions to start being displayed again since we took away their right to when we started our SuggestionSession.

Reply:
so, generally speaking, cmments like this in the PR are worth putting into the code, because the code isn't clear as to why that's going on.